app/models/metric/helper.rb

```diff
@@ -165,4 +165,14 @@ def self.get_time_range_from_offset(start_offset, end_offset = nil, options = {}
 
     return start_time, end_time
   end
+
+  def self.get_time_interval(obj, attrs, ts = nil)
+    ts ||= attrs[:timestamp]
+    state = obj.vim_performance_state_for_ts(ts)
+
+    ts = Time.parse(ts).utc if ts.kind_of?(String)
```

Comment:
@chessbyte moved it _before_ 
the reason it looks strange is because it used to be:
_start_time..end_time_, and after some edits it ended up _start_time..ts_
maybe adding back the _end_time_ variable will make it less strange ?